aten/src/ATen/native/cuda/Indexing.cu

```diff
@@ -503,10 +503,10 @@ void index_put_with_sort_quantized(Tensor & self, const c10::List<c10::optional<
         float inv_scale = 1.0f / static_cast<float>(scale);
 
         indexing_backward_kernel_quantized<scalar_t, UNROLL><<<grid, block, 0, stream>>>(
-          sorted_indices.data_ptr<int64_t>(),
-          orig_indices.data_ptr<int64_t>(),
-          expandedValue.data_ptr<float>(),
-          src_.data_ptr<scalar_t>(),
+          sorted_indices.const_data_ptr<int64_t>(),
+          orig_indices.const_data_ptr<int64_t>(),
+          expandedValue.const_data_ptr<float>(),
+          src_.mutable_data_ptr<scalar_t>(),
```

Comment:
Is this... mutating through a `contiguous()` call? wut

Reply:
Any action required here? A side benefit of all this is that we can address things that seem a bit off, by either filing an issue, adding a comment, or fixing in place.

Reply:
You should add this to your internal list because this may trigger your warning (or maybe not, if the original empty tensor is dead at this point.)